crates/payload/builder/src/service.rs

```diff
@@ -203,7 +204,9 @@ where
                         let id = attr.payload_id();
                         let mut res = Ok(id);
 
-                        if !this.contains_payload(id) {
+                        if this.contains_payload(id) {
```

Comment:
do we need to increase the deadline in this case?

Reply:
no how long it's running should be managed by the job itself, default is 12s (or 1 blocktime)